src/handlers/motions/motionStaked/helpers.ts

```diff
@@ -182,3 +182,71 @@ export const getUserMinStake = (
     .div(BigNumber.from(10).pow(36))
     .toString();
 };
+
+interface Props {
+  motionData: MotionData,
```

Comment:
Commas should be semi-colons

Reply:
Are you using prettier? It catches things like this automatically

Reply:
@willm30 I fixed the formatting issues. I needed to create a .prettierrc file to get prettier to run correctly, how are you using prettier in the block ingestor?

Reply:
I have it installed in vsc

Reply:
I had it installed in vsc but not sure why it wasn't working. Thanks again for pointing it out.